docs/docs/general/06-Tutorials/zip-images-upload-googledrive-windows.md

```diff
@@ -9,8 +9,11 @@ This tutorial will show you how to zip files on your file system and then upload
 <div class="cookieconsent-optout-marketing">
   Please <a href="javascript:Cookiebot.renew()">accept marketing-cookies</a> to watch this video.
 </div>
+<div class="cookieconsent-optin-marketing">
+  <a href="javascript:rerunCookieConsentScripts(0)">Please click to watch this video.</a>
+</div>
+<iframe onload="rerunCookieConsentScripts(0);" width="560" height="315" data-cookieblock-src="https://www.youtube-nocookie.com/embed/i3M5SPYQTKI&embedded=true" data-cookieconsent="marketing" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Didn't you say that the `onload` does not work? 😄

Reply:
Will be removed 😉